src/commands/package.js

```diff
@@ -116,3 +112,102 @@ export default asyncCommand({
 	}
 });
 
+
+function getZimletLabelDescription() {
+	const cwd = process.cwd();
+	const intlDir = path.resolve(cwd, 'src', 'intl');
+
+	try {
+		const content = JSON.parse(
+			fs.readFileSync(path.resolve(intlDir, 'en_US.json'))
+		);
+
+		if (!(content.zimlet?.label && content.zimlet?.description)) {
+			throw new Error('label or description not found');
+		}
+		return {
+			label: content.zimlet.label,
+			description: content.zimlet.description
+		};
+	}
+	catch (err) {
+		warn(`Failed to read src/intl/en_US.json file: ${err}. \nMake sure below content present in src/intl/en_US.json file \n
+		"zimlet": {
+			"label": "<zimlet label>",
+			"description": "<zimlet description>"
+		} \nAs a fallback we are using strings from package.json which is deprecated.\n`);
+		return null;
+	}
+
+}
+
+function createLocalizationFiles(zimletName, label, description) {
+	const cwd = process.cwd();
+	const intlDir = path.resolve(cwd, 'src', 'intl');
+	let intlFiles;
+
+	// Add properties file for description and label
+	// As admin console requires properties file to display label and desciption
+	createPropertyFile(zimletName, label, description);
+
+	try {
+		intlFiles = fs.readdirSync(intlDir);
+	}
+	catch (err) {
+		console.error(`Failed to read ${intlDir}: ${err}`, 1);
+	}
+
+	intlFiles
+		.filter((file) => file.match(/\.json$/))
+		.forEach((intl) => {
+			try {
+				const content = JSON.parse(
+					fs.readFileSync(path.resolve(intlDir, intl))
+				);
+				const propertiesFileName = `${zimletName}_${intl.replace(
+					'.json',
+					''
+				)}`;
+
+				let labelValue = content.zimlet?.label || '';
+				let descriptionValue = content.zimlet?.description || '';
+
+				if (!intl.includes('en_US')) {
```

Comment:
why not for english language?

Reply:
For other than english property file, we are converting value of label and description into unicode which is not require for en_US property file.